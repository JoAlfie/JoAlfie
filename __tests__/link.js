import React from "react"
import renderer from "react-test-renderer"
import Link from "../src/components/link"

describe("Link", () => {
	it("renders Gatsby internal link correctly", () => {
		const tree = renderer.create(
			<Link
				to="/test"
				activeClassName="activeclass"
				partiallyActive="partiallyactive"
			>
				Gatsby link
			</Link>
		)
		expect(tree).toMatchSnapshot()
	})
})

describe("Link", () => {
	it("renders external link correctly", () => {
		const tree = renderer.create(
			<Link to="https://google.com">Google link</Link>
		)
		expect(tree).toMatchSnapshot()
	})
})
