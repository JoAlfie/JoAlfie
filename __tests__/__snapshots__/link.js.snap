// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders Gatsby internal link correctly 1`] = `
<a
  href="/test"
>
  Gatsby link
</a>
`;

exports[`Link renders external link correctly 1`] = `
<a
  href="https://google.com"
>
  Google link
</a>
`;
